pkg/apis/pipeline/v1beta1/pipelinerun_types.go

```diff
@@ -416,6 +416,13 @@ type PipelineRunStatusFields struct {
 	// CompletionTime is the time the PipelineRun completed.
 	CompletionTime *metav1.Time `json:"completionTime,omitempty"`
 
+	// TaskRuns is a map of PipelineRunTaskRunStatus with the taskRun name as the key.
+	// Deprecated: use ChildReferences instead. As of v0.45.0, this field is no
+	// longer populated and is only included for backwards compatibility with
+	// older server versions.
+	// +optional
+	TaskRuns map[string]*PipelineRunTaskRunStatus `json:"taskRuns,omitempty"`
```

Comment:
if we're bringing back TaskRuns we should bring back Runs as well

Reply:
Done!